resource-managers/kubernetes/integration-tests/scripts/setup-integration-test-env.sh

```diff
@@ -59,6 +60,10 @@ while (( "$#" )); do
       SPARK_TGZ="$2"
       shift
       ;;
+    --docker-file)
```

Comment:
Do we need to update `resource-managers/kubernetes/integration-tests/README.md`?